spacemacs/packages.el

```diff
@@ -2492,7 +2492,7 @@ displayed in the mode-line.")
 
 (defun spacemacs/init-volatile-highlights ()
   (use-package volatile-highlights
-    :init
+    :config
```

Comment:
From the readme:

> Use the :init keyword to do some stuff to initialize foo. If loading was deferred, the code is run immediately; otherwise the package is required before running the code.

So it should be safe to do it in `:init` without a `:defer`.

I'm starting to wonder if I should make a light `use-package` version for spacemacs with a cleaner design, `use-package` grew organically and it is arrived at the point where some feature proposal get stuck because nobody has a clear understanding of how things will be executed in the end.

For more info see: https://github.com/jwiegley/use-package/issues/161